frontend/src/global_styles/content/_attributes_key_value.sass

```diff
@@ -31,11 +31,6 @@
   // flex-wrap: wrap
   font-size: 0.875rem
   margin-bottom: 0.1875rem
-  flex-basis: 400px
-  flex-grow: 1
-  flex-shrink: 1
-  min-width: calc(50% - 3rem)
-  max-width: calc(100% - 3rem)
   margin: 0 1.5rem
```

Comment:
As the attributes are now all below each other we can get rid of this extra margin and the magical calculation done here: https://github.com/opf/openproject/blob/a705bc530f4a2bd87d0f8e03ae4244b9b8b82bca/frontend/src/global_styles/content/_attributes_group.sass#L85C1-L90C1